### PR TITLE
Add file for Fraction class

### PR DESCRIPTION
**Pull request by lpugin**
* No change expected

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]

File: bindings/iOS/all.h [hunk 1]
```diff
@@ -184,6 +190,7 @@
 #import <VerovioFramework/positioninterface.h>
 #import <VerovioFramework/preparedatafunctor.h>
 #import <VerovioFramework/proport.h>
+#import <VerovioFramework/quilisma.h>
```
Comment (ahankinson) [inline bindings/iOS/all.h:193]: Just checking that these are supposed to be here? They don't seem like they're immediately relevant to this PR?

File: include/vrv/fraction.h [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        faction.h
```
Comment (ahankinson) [inline include/vrv/fraction.h:2]:
```suggestion
// Name:        fraction.h
```

File: bindings/iOS/all.h (hunk 1 above)
Comment (lpugin) [reply bindings/iOS/all.h:193]: It happens for the `iOS` framework to be outdated, and it was the case here. In any case, it is always updated during releases, but I am also trying to keep it updated whenever new files are added.

**Review by ahankinson — APPROVED** [review]
Just a couple questions, but otherwise looks good.

File: bindings/iOS/all.h (hunk 1 above)
Comment (ahankinson) [reply bindings/iOS/all.h:193]: OK!

**Pull request merged**